club/models.py

```diff
@@ -0,0 +1,73 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class Member(AbstractUser):
+    email = models.EmailField(unique=True)
+    type = models.CharField(max_length=20)
+    address = models.CharField(max_length=100)
+    home_phone = models.CharField(max_length=20)
```

Comment:
Could use a plugin: https://pypi.org/project/django-phone-field/ but might also be overkill, I am ok with a charfield for now

Reply:
I'll try it, something additional to discover

Reply:
I am happy to keep validation lose for now though, it's less urgent for the greater scope, let me just issue this.